src/common/modules/navigations/gnb/modules/gnb-noti/modules/GNBNotificationsTab.vue

```diff
@@ -100,6 +152,20 @@ export default {
             items: [] as NotificationItem[],
             proxyCount: useProxyValue('count', props, emit),
             pageStart: 1,
+            //
+            modalVisible: false,
+            selectedItem: {} as NotificationItem,
+            definitionFields: computed(() => state.selectedItem?.message?.tags.map(d => ({
+                name: d.key, label: d.key,
+            }))),
+            definitionData: computed(() => {
+                const result = {};
+                // eslint-disable-next-line no-unused-expressions
+                state.selectedItem?.message?.tags.forEach((d) => {
```

Comment:
꼭 필요한 혹은 임시적인 상황이 아니면 eslint disabled 코멘트는 최대한 사용 자제했으면 합니다.
아래처럼 바꾸면 아마 문제없을 거라고 봅니다!
```suggestion
                if (Array.isArray(state.selectedItem?.message?.tags)) state.selectedItem.message.tags.forEach((d) => {
```

Reply:
오 이거 어떻게 해결하나 했는데...넘 감사합니다!